src/robusta/core/playbooks/prometheus_enrichment_utils.py

```diff
@@ -220,6 +228,14 @@ def create_resource_enrichment(
         if title_override
         else f"{resource_type.name} {values_format_text} for this {item_type.name.lower()}"
     )
+
+    # NOTE: Some queries do not prodice automatic labels, so we need to override them
+    # Numerical index is the number of the series in the chart to override (excluding lines)
+    chart_labels_overrides = {
+        (ResourceChartResourceType.Memory, ResourceChartItemType.Container): {
+            0: "Memory Usage",
```

Comment:
I understand the comment on the index, but it's a little confusing.
What's the scenario where a query returns multiple series?
If we wanted to run the container memory query without filter on the container. 
Will we get series per container?  I'm trying to think if this solution works if the number of the returned series is unknown.
Can there be a different override label for different series on the result, for a single query?